examples/restart/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Globalization;
+using System.Text.Json;
+using Json.Patch;
+using k8s;
+using k8s.Models;
+
+double ConvertToUnixTimestamp(DateTime date)
+{
+    var origin = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
+    var diff = date.ToUniversalTime() - origin;
+    return Math.Floor(diff.TotalSeconds);
+}
+
+async Task RestartDamonsetAsync(string name,string @namespace)
+{
+    var config = KubernetesClientConfiguration.BuildConfigFromConfigFile();
+    IKubernetes client = new Kubernetes(config);
+    var daemonSet = await client.ReadNamespacedDaemonSetAsync(name, @namespace);
+    var options = new JsonSerializerOptions
+    {
+        PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+        WriteIndented = true
+    };
+    var old = JsonSerializer.SerializeToDocument(daemonSet,options);
+
+    var restart = new Dictionary<string, string>(daemonSet.Spec.Template.Metadata.Annotations)
+    {
+        ["date"] = ConvertToUnixTimestamp(DateTime.UtcNow).ToString(CultureInfo.InvariantCulture)
+    };
+
+    daemonSet.Spec.Template.Metadata.Annotations = restart;
+
+    var expected = JsonSerializer.SerializeToDocument(daemonSet);
+
+    var patch = old.CreatePatch(expected);
+    await client.PatchNamespacedDaemonSetAsync(new V1Patch(patch, V1Patch.PatchType.JsonPatch), name, @namespace);
+}
+
+async Task RestartDeploymentAsync(string name,string @namespace)
+{
+    var config = KubernetesClientConfiguration.BuildConfigFromConfigFile();
```

Comment:
This code is duplicated in a bunch of places, perhaps it makes more sense to pass the client to the `RestartDeploymentAsync` function?